test/server/SimpleServer.js

```diff
@@ -99,6 +101,10 @@ class SimpleServer {
     this._auths.set(path, {username, password});
   }
 
+  setCSP() {
```

Comment:
I'd like this to take a boolean: `setCSP(true)`. Otherwise I will use setCSP(false) and become very confused as to why it doesn't work 😀

Reply:
Thanks! I applied your comment [here](https://github.com/GoogleChrome/puppeteer/pull/1624/files#diff-38e89beafd599cb655dc914a04f642feR104)!